README.md

```diff
@@ -1 +1,174 @@
-# Capstone-project
\ No newline at end of file
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Festival_Amani_Website] <a name="about-project"></a>
+
+**[Festival_Amani_Website]** is a project that will allow people to what the festival amani provide in Goma town and the date of the next session of the Amani festival.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">Html</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+    <li><a href="https://www.w3schools.com/javascript/">Javascript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Know the purpose of Amani Festival]**
+- **[To reachout the world over]**
+- **[Understand what happen in the east of Congo]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://kaghenimbale.github.io/Capstone-project/)
+
+<!-- VIDEO DEMO -->
+
+- [Video explanation of the project](https://www.awesomescreenshot.com/video/14637069))
```

Comment:
- [ ] While you add the required video to your README file as requested by the previous reviewer :+1: the link seems to be broken or maybe it cannot be accessed because it is not public. Kindly try to fix this link error :+1: 

![Screenshot from 2023-02-08 07-39-55](https://user-images.githubusercontent.com/105117832/217533502-090047c9-4dd5-40c5-b419-ef0836a697e1.png)